src/sentry/static/sentry/app/components/searchBar.tsx

```diff
@@ -103,11 +105,15 @@ class SearchBar extends React.PureComponent<Props, State> {
             />
             <StyledIconSearch className="search-input-icon" size="sm" color="gray500" />
             {this.state.query !== this.props.defaultQuery && (
-              <div>
-                <a className="search-clear-form" onClick={this.clearSearch}>
-                  <IconClose />
-                </a>
-              </div>
+              <SearchClearForm
+                type="button"
+                className="search-clear-form"
```

Comment:
Can we get rid of this className too?

Reply:
I looked into that, but I wasn't sure I wanted to refactor this usage https://github.com/getsentry/sentry/blob/9b5e467c6e5d729388f0e08fb0eeec4bd401f3fb/src/sentry/static/sentry/app/components/events/interfaces/breadcrumbs/index.tsx#L439